server/src/test/java/keywhiz/KeywhizConfigTest.java

```diff
@@ -0,0 +1,60 @@
+package keywhiz;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import io.dropwizard.configuration.ConfigurationException;
+import io.dropwizard.configuration.YamlConfigurationFactory;
+import io.dropwizard.setup.Bootstrap;
+import java.io.File;
+import java.io.IOException;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class KeywhizConfigTest {
+  private static final String PROPERTY_PREFIX = "dw";
+  private static final String TEST_CONFIGS_PREFIX = "configs/";
+
+  @Test
+  public void parsesMaximumSecretLength() {
+    KeywhizConfig config = loadConfig("with-secret-size-limit-123456.yaml");
+    assertEquals(Long.valueOf(123456), config.getMaximumSecretSizeInBytes());
+  }
+
+  @Test
+  public void missingMaximumSecretLengthIsNull() {
+    KeywhizConfig config = loadConfig("without-secret-size-limit.yaml");
+    assertNull(config.getMaximumSecretSizeInBytes());
+  }
+
+  private static KeywhizConfig loadConfig(String resource) {
+    Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+    ObjectMapper objectMapper = createObjectMapper();
+    File yamlFile = new File(Resources.getResource(TEST_CONFIGS_PREFIX + resource).getFile());
```

Comment:
I created a new folder for test configs, similar to how there are already folders for other test resource types. The idea was to be able to easily load configs with specific properties and run tests against them.